valsamoggia.ninux.org/vs-ninux-generic/etc/config/lime-defaults

```diff
@@ -0,0 +1,37 @@
+config lime system
+	option hostname 'LiMe-%M4%M5%M6'
+	option domain 'valsamoggia.ninux.org'
+	option keep_on_upgrade 'libremesh base-files-essential /etc/sysupgrade.conf'
+
+config lime network
+	option primary_interface 'eth0'
+	option main_ipv4_address '10.%N1.128.0/16/17'
+	option anygw_dhcp_start '5120'
+	option anygw_dhcp_limit '27648'
+	option main_ipv6_address '2a00:1508:0a%N1:%N200::/64'
+	list protocols ieee80211s
+	list protocols lan
+	list protocols anygw
+	list protocols batadv:%N1
+	list protocols babeld:0
```

Comment:
@digitigrafo be especially careful using babeld without VLAN, the support for that setup is quite recent, and AFAIK not battle tested.
@ilario did all the related PR got already merged? did you tested it with a bunch of different devices and situations? in particular with cable links on WAN+WAN and LAN+LAN ?
Cheers!